### PR TITLE
Init version 23.10.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by NvTimLiu**
bump up JNI version to 23.10.0

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

**Review by pxLi — APPROVED** [review]
LGTM. We should wait for a clean CI pass before merge, thanks

**Comment by pxLi** [comment]
oh neat pass! Going to merge this, we can start preparing new artifacts

**Pull request merged**
